.github/workflows/fetch_translations.yml

```diff
@@ -0,0 +1,34 @@
+name: Fetch Lokalize translations
+on:
+  workflow_dispatch:
+  schedule:
+    # Every Monday at 10 AM East Coast time / 7 AM West Coast time / 2 PM UTC
+    - cron: "0 14 * * 1"
+jobs:
+  fetch-lokalize-translations:
+    runs-on: macos-latest
+    steps:
+      # Install Brew so that we can use the Lokalize CLI
+      - run: brew tap lokalise/cli-2
+      - run: brew install lokalise2
+
+      # Check out the current state
+      - uses: actions/checkout@v3
+
+      # Fetch translations
+      # Note: The environment variable points to tillh's Lokalize username and token.
+      - name: Run build script
+        run: cd scripts && ./localize.sh
+        shell: bash
+        env:
+          LOKALISE_API_TOKEN: ${{ secrets.LOKALISE_API_TOKEN }}
+
+      # Create the pull request
+      - name: Create Pull Request
```

Comment:
so we'll still need to approve and merge it manually right?

Reply:
Yes, that’s right. We need to approve _any_ change for PCI compliance.